test/actions/pulp3/orchestration/rpm_remove_units_test.rb

```diff
@@ -12,6 +12,7 @@ def setup
 
       create_and_sync_repo(@repo, @master)
       @repo.reload
+      @rpm = Katello::Rpm.all.where(filename: "frog-0.1-1.noarch.rpm")[0]
```

Comment:
Few nitpicks:  i'd probably look in the repo for the rpm instead of querying it across Rpm.all (i think that'd be less likely to fail in the future).  I'd also use .find_by instead of .where()[0]